vector/src/main/java/im/vector/app/features/spaces/create/CreateSpaceViewModelTask.kt

```diff
@@ -57,7 +58,8 @@ class CreateSpaceViewModelTask @Inject constructor(
 
     override suspend fun execute(params: CreateSpaceTaskParams): CreateSpaceTaskResult {
         val spaceID = try {
-            session.spaceService().createSpace(params.spaceName, params.spaceTopic, params.spaceAvatar, params.isPublic)
+            session.spaceService().createSpace(params.spaceName, params.spaceTopic, params.spaceAvatar,
+                    params.isPublic, params.spaceAlias)
```

Comment:
maybe put one param per line and add the param name for clarity

Reply:
done